src/main/java/com/salesforce/cdp/queryservice/util/Constants.java

```diff
@@ -33,6 +33,19 @@
     public static final String DRIVER_NAME = "QueryService-jdbc";
     public static final String DRIVER_VERSION = "1.0";
 
+    // Common client id and secret information
+    public static final String STMPA_SERVER_URL = "stmpa.stm.salesforce.com";
+    public static final String STMPA_DEFAULT_CLIENT_ID = "3MVG9Iu66FKeHhIMTw4_fbTKdbfgYVQXSsSJ6jOMZWGwgJ0RVhaGO3_RXBTxmWUHholNWtSgpa6nmjNeOvdBX";
+    public static final String STMPA_DEFAULT_CLIENT_SECRET = "9662F811A0702D3FE62BCF2AF3A51D06F38E1D0FA1573E5D17301C9FA4B216BB";
+
+    public static final String STMPB_SERVER_URL = "stmpb.stm.salesforce.com";
+    public static final String STMPB_DEFAULT_CLIENT_ID = "3MVG9VeAQy5y3BQVJqaUbFmV5jd8imcck2K5idmrTTGocSu9qZZ6qkbuEkxECKVYwmzm3WgvxkujqsxZDcBpL";
```

Comment:
Client Id of STMPB is 3MVG9Iu66FKeHhIPHKRhbohYVNiUUHCt2h_HeUwYwzEdrn4unY9uMKKdfYeGnb19ngLq6T6Q8hwEHVexROWav

Reply:
fixed. thanks for catching this.